content/actions/using-workflows/reusing-workflows.md

```diff
@@ -112,11 +112,10 @@ You can define inputs and secrets, which can be passed from the caller workflow
      reusable_workflow_job:
        runs-on: ubuntu-latest
        environment: production
-       steps:
-         - uses: ./.github/workflows/my-action
-           with:
-             username: ${{ inputs.username }}
-             token: ${{ secrets.envPAT }}
+       uses: ./.github/workflows/my-action
+       with:
+         username: ${{ inputs.username }}
+         token: ${{ secrets.envPAT }}
```

Comment:
👋 I'm not sure this change is correct. This example is for the reusable workflow itself, not the caller workflow. Reusable workflows do need steps.

The caller workflow example is shown in the next step in the procedure.

@ethomson can you confirm?

We could make this clearer with the lead-in sentence to this step in the procedure. Instead of:

```
1. Reference the input or secret in the reusable workflow.
```

we could say,

```
1. In the reusable workflow, reference the input or secret that you defined in the `on` key in the previous step.
```

Reply:
Oh! Thanks @lucascosti, I misunderstood this. Thanks for the careful review!

Reply:
> Oh! Thanks @lucascosti, I misunderstood this. Thanks for the careful review!

No worries! 🙂 

@nowsprinting Did you want to revert your change and implement the clarification suggestion above? Otherwise, we'll close this PR. 🙇‍♂️

Reply:
@lucascosti Thank you for your review. And I’m sorry. I misread it.
I try to revert and fix sentence via your recommendation.

Reply:
Done!
plz re-review!